doc/how_to_rename_chroot.rst

```diff
@@ -0,0 +1,50 @@
+.. _how_to_rename_chroot:
+
+How to rename a chroot
+================================
+
+In the openEuler copr instance, we unfortunately need to rename a chroot.
+
+The operations need to be done can be described as:
+
+* [backend]: symlinks all the old chroot dir in resultdir to the new one
+* [frontend]: change the os_release/os_version in frontend table mock_chroot
+* [builder]: add new mock chroot configs to all builder
+
+.. note::
+
+    After these operations are complete, a new chroot will be displayed at all frontend pages, and all the old repo config files should be still usable.
+
+To be more precisely, here are some operation details:
+
+*Suppose you have a chroot named* ``foo-bar-x86_64`` *and want to rename it to* ``new-foo_bar-x86_64``
+
+For backend
+--------------
+Login as copr user to backend node, and run:
+
+.. code-block:: console
+
+    $ /usr/bin/copr-rename-chroot --real-run  --pair PAIRS foo-bar-x86_64:new-foo_bar-x86_64
```

Comment:
The `PAIRS` argument seems like a leftover.

Reply:
> The `PAIRS` argument seems like a leftover.

Good catch thanks!